compressor/management/commands/compress.py

```diff
@@ -250,7 +250,15 @@ def compress(self, log=None, **options):
                     except Exception as e:
                         raise CommandError("An error occurred during rendering %s: "
                                            "%s" % (template.template_name, smart_text(e)))
-                    offline_manifest[key] = result
+
+                    if settings.COMPRESS_OFFLINE_USE_URL_PLACEHOLDER:
+                        # Replace settings.COMPRESS_URL with a placeholder
+                        offline_manifest[key] = result.replace(
+                            settings.COMPRESS_URL, settings.COMPRESS_URL_PLACEHOLDER
+                        )
+                    else:
+                        offline_manifest[key] = result
+
```

Comment:
Yes, I see what you mean: writing the manifest, then reading, then writing again. 
I'm not sure though if it'll give the same effect if I'll move it to `.handle()`...